idmapping-rest/src/main/java/org/uniprot/api/idmapping/service/job/PIRJobTask.java

```diff
@@ -47,8 +46,8 @@ protected IdMappingResult processTask(IdMappingJob job) {
                     && Objects.nonNull(job.getIdMappingRequest())
                     && IdMappingFieldConfig.UNIPROTKB_STR.equals(
                             job.getIdMappingRequest().getTo())) {
-
-                Integer obsoleteUniProtCount = getObsoleteUniProtEntryCount(result);
+                int batchSize = 100_000;
```

Comment:
This is very specific use case unlike Voldemort. Voldemort is the most common use case and we need to tune it more frequently. But for this externalising the value will not help much since we will hardly tune it.